schoolbot/cogs/school/timetable.py

```diff
@@ -0,0 +1,139 @@
+                if isinstance(e, DataNotFound):
+                    return await msg.edit(
+                        embed=discord.Embed(title="정보가 없습니다. 확인하신후 다시 요청하세요")
+                    )
+                else:
+                    # str(uuid.uuid1())
+                    # traceback.format_exc()
+                    return await msg.edit(
+                        embed=discord.Embed(title="알수없는 오류입니다", description=f"{e}")
+                    )
+            if len(scinfo.data) > 1:
+                school_name_list = [
+                    school_name["SCHUL_NM"] for school_name in scinfo.data
+                ]
+                school_name_list_with_num = [
+                    str(index) + ". " + school_names
+                    for index, school_names in enumerate(school_name_list, 1)
+                ]
+                await msg.edit(
+                    embed=discord.Embed(
+                        title="여러개의 검색결과입니다. 다음중 선택해주세요.",
+                        description="\n".join(school_name_list_with_num),
+                    )
+                )
+
+                try:
+                    response = await self.bot.wait_for(
+                        "message",
+                        check=lambda m: m.author == ctx.author
+                        and m.channel == ctx.channel,
+                        timeout=30,
+                    )
+                except asyncio.TimeoutError:
+                    return await msg.edit(
+                        embed=discord.Embed(title="시간 초과입니다. 처음부터 다시 시도해주세요.")
+                    )
+                else:
+                    fetch_msg = await ctx.fetch_message(response.id)
+                    try:
+                        num = int(fetch_msg.content) - 1
+                    except ValueError:
+                        return await msg.edit(
+                            embed=discord.Embed(title="잘못된값을 주셨습니다. 처음부터 다시 시도해주세요.")
+                        )
+                    else:
+                        choice = scinfo.data[num]
+                        AE = choice["ATPT_OFCDC_SC_CODE"]
+                        SE = choice["SD_SCHUL_CODE"]
+                        SN = choice["SCHUL_NM"]
+            else:
+                choice = scinfo.data[0]
+                AE = choice["ATPT_OFCDC_SC_CODE"]
+                SE = choice["SD_SCHUL_CODE"]
+                SN = choice["SCHUL_NM"]
+        else:
+            # 대충 여따가 쿼리문 적으면 된다는 주석
+            # AE = 대충 교육청코드
+            # SE = 대충 표준학교코드
+            # scclass = 대충 초 중 고 고르는거
+            return await msg.edit(
+                embed=discord.Embed(title="학교명을 입력해주세요")
+            )  # 쿼리문 쓰고 지워도 되는거
+
+        # if scclass:
+        #    pass
+        if "초등학교" in SN:
+            scclass = "els"
+        elif "중학교" in SN:
+            scclass = "mis"
+        elif "고등학교" in SN:
+            return await msg.edit(
+                embed=discord.Embed(title="죄송합니다. 현재 고등학교는 지원하지않습니다.")  # 고등학교 지원할때 빼면됨
+            )
```

Comment:
`scclass` can be undefined

```suggestion
        if "초등학교" in SN:
            scclass = "els"
        elif "중학교" in SN:
            scclass = "mis"
        elif "고등학교" in SN:
            return await msg.edit(
                embed=discord.Embed(title="죄송합니다. 현재 고등학교는 지원하지않습니다.")  # 고등학교 지원할때 빼면됨
            )
        else:
            return await msg.edit(
                embed=discord.Embed(title="죄송합니다. 지원하지 않는 학교입니다.")
            )
```